sdk/ml/azure-ai-ml/tests/dsl/unittests/test_dsl_pipeline_with_specific_nodes.py

```diff
@@ -1362,9 +1362,9 @@ def pipeline(iris_data):
             }
 
         pipeline1 = pipeline(iris_data)
-        with pytest.raises(ValidationException) as ve:
-            pipeline1._to_rest_object().as_dict()
-            assert ve.message == "Should not specify min or max executors when dynamic allocation is disabled."
+        result = pipeline1._validate()
+        assert 'jobs.node.conf' in result.error_messages and result.error_messages['jobs.node.conf'] == \
```

Comment:
this test hasn't covered your change, right? Newly appended error is with a yaml path of `*`

Reply:
No, I doesn't remove _validate_spark_configurations in spark.py since I remove this
![image](https://user-images.githubusercontent.com/26239730/223905194-61e90b7e-6970-4c18-94e8-442a389abf52.png)

Reply:
Also, not sure if you have added such e2e tests. in e2e tests you may 
```
with pytest.raises(ValidationException, match="xxx")
  ml_client.create_or_update(pipeline1)
```

Reply:
should I remove this statement in common validation function added by spark team and modify standalone code?

Reply:
Yes, I think refactor common validation function added by spark team is the best practice. And you may involve me when you ask for their approval.